core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -288,7 +288,7 @@ def score_with_oof(self, y, sample_weight=None):
     # TODO: Augment to generate OOF after shuffling each column in X (Batching), this is the fastest way.
     # TODO: v0.1 Reduce logging clutter during OOF importance calculation (Currently logs separately for each child)
     # Generates OOF predictions from pre-trained bagged models, assuming X and y are in the same row order as used in .fit(X, y)
-    def compute_feature_importance(self, X, y, features=None, is_oof=True, time_limit=None, silent=False, **kwargs) -> pd.DataFrame:
+    def compute_feature_importance(self, X, y, features=None, is_oof=False, time_limit=None, silent=False, **kwargs) -> pd.DataFrame:
```

Comment:
why change default here?

Reply:
default was always overwritten outside, and because all other models effectively have it set to False (since they don't have oof), so this aligns default functionality with non-bagged models.